test/widgets/content_test.dart

```diff
@@ -154,6 +155,43 @@ void main() {
     });
   }
 
+  /// To check font weight, set up tests with and without bold-text setting.
```

Comment:
The "To check" here reads like it's an instruction: "If you want to check font weight, set up tests thus-and-so."

A direct fix for that would be:
> Set up tests with and without bold-text setting, to check font weight.

But I feel like what I want in the summary line is more of a description of what the said tests will test. Here's a version:

```suggestion
  /// Test the font weight found by [styleFinder] in the rendering of [content].
  ///
  /// The weight will be expected to be [expectedWght] when the system
  /// bold-text setting is not set, and to vary appropriately when it is set.
```